tembo-stacks/src/apps/ai.yaml

```diff
@@ -0,0 +1,63 @@
+name: !ai-proxy
+appServices:
+  - image: quay.io/tembo/ai-proxy:2024.06.03
+    name: ai-proxy
+    resources:
+      limits:
+        cpu: 100m
+        memory: 150Mi
+      requests:
+        cpu: 10m
+        memory: 150Mi
+    routing:
+    - port: 8080
+    storage:
+      volumeMounts:
+      - mountPath: /etc/nginx/conf.d
+        name: empty-dir
+      - mountPath: /var/cache/nginx
+        name: empty-dir
+      - mountPath: /var/run
+        name: empty-dir
+      volumes:
+      - emptyDir: {}
+        name: empty-dir
+trunk_installs:
+  - name: pg_cron
+    version: 1.6.2
+  - name: pgmq
+    version: 1.3.3
+  - name: pgvector
+    version: 0.7.1
+  - name: vectorize
+    version: 0.16.0
+extensions:
+  - name: pg_cron
+    locations:
+      - database: postgres
+        enabled: true
+        version: 1.6.2
+  - name: pgmq
+    locations:
+      - database: postgres
+        enabled: true
+        version: 1.3.3
+  - name: vector
+    locations:
+      - database: postgres
+        enabled: true
+        version: 0.7.1
+  - name: vectorize
+    locations:
+      - database: postgres
+        enabled: true
+        version: 0.16.0
+postgres_config:
+  - name: cron.host
+    value: /controller/run
+  - name: vectorize.host
+    value: postgresql:///postgres?host=/controller/run
+  - name: vectorize.tembo_service_url
+    value: http://${NAMESPACE}-ai-proxy:8080/v1
+  - name: vectorize.tembo_jwt
+    value: placeholder
```

Comment:
What does the `value: placeholder` mean here?

Reply:
setting it to a "not null" for Tembo cloud users.